ui/lib/utils/port.ts

```
@@ -9,119 +9,119 @@
+    // Production mode: use relative URL for API calls
+    return '/api'
+  }
 }
 
 /**
  * Get the WebSocket URL for real-time connections
  */
-export function getWebSocketUrl(path: string = ""): string {
-	const config = getPortConfig();
-	const cleanPath = path.startsWith("/") ? path : `/${path}`;
+export function getWebSocketUrl(path: string = ''): string {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)